src/utils/rest-api.js

```diff
@@ -483,6 +483,24 @@ export function getNameCandidate(directoryUuid, elementName, type) {
     });
 }
 
+/**
+ * Retrieves the original name of a case using its UUID.
+ * @param {string} caseUuid - The UUID of the element.
+ * @returns {Promise<string|boolean>} - A promise that resolves to the original name of the case if found, or false if not found.
+ */
+export function getCaseOriginalName(elementUuid) {
```

Comment:
sorry but JsDoc above is not good : caseUuid vs elementUuid

Reply:
sorry , i corrected it